src/Operations.jl

```diff
@@ -1333,6 +1333,7 @@ function test(ctx::Context, pkgs::Vector{PackageSpec}; coverage=false)
             --color=$(Base.have_color ? "yes" : "no")
             --compiled-modules=$(Bool(Base.JLOptions().use_compiled_modules) ? "yes" : "no")
             --check-bounds=yes
+            --inline=$(inline ? "yes" : "no")
```

Comment:
Base this on `Base.JLOptions().can_inline`, similar to other options?

Reply:
I thought about doing that, too. One drawback of that would be that all packages affected by the problem at hand would have to be updated to explicitly add `--inline=no` to their test script(s), and of course also the default Julia scripts for Travis CI and AppVeyor would need to be adapted. I.e. far more people would have to change far more code.

That said, of course I can change it to do that *shrug*